Makefile

```diff
@@ -343,6 +343,10 @@ SWIFT_REPO       ?= https://github.com/openstack-k8s-operators/swift-operator.gi
 SWIFT_BRANCH     ?= main
 SWIFT            ?= config/samples/swift_v1beta1_swift.yaml
 SWIFT_CR         ?= ${OPERATOR_BASE_DIR}/swift-operator/${SWIFT}
+SWIFTAPI_DEPL_IMG  ?= unused
```

Comment:
Is this used anywhere?

Reply:
no its not, do we need it? hmmm

Reply:
I guess we don't need images for swift, but other do require images?  Is that the difference?  If so, I will remove this line.

Reply:
We can always add it later if we need it for something